knowledge-base/spreadsheet-hide-toolbar-formula-sheet-bar.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Hide Toolbar, Formula Bar, or Sheet Bar in Spreadsheet
+description: Learn how to hide the toolbar, formula bar, and sheet bar in the Blazor Spreadsheet component.
+type: how-to
+page_title: How to Change the Visibility of Spreadsheet Parts in Blazor
+slug: spreadsheet-kb-hide-toolbar-formula-sheet-bar
+tags: spreadsheet, blazor, toolbar, formula bar, sheet bar, customization, css, visibility
+res_type: kb
+ticketid: 1676073
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Spreadsheet for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+I want to display only the spreadsheet portion and hide the toolbar, formula bar, and sheet bars in the Spreadsheet component.
+
+## Solution
+
+To change the visibility of the toolbar, formula bar, and sheet bar in the Spreadsheet for Blazor, apply conditional CSS classes to hide the respective components.
```

Comment:
Consider adding a note that this approach is acceptable only if the app can trust its users to not show the tools back through the browser console.